example/lib/src/file_picker_demo.dart

```diff
@@ -33,11 +55,32 @@ class _FilePickerDemoState extends State<FilePickerDemo> {
         .addListener(() => _extension = _fileExtensionController.text);
   }
 
+  Widget _buildFilePickerResultsWidget(
+      {required int itemCount,
+      required Widget Function(BuildContext, int) itemBuilder}) {
```

Comment:
Missing comma, this results in a weird formatting diff.

```suggestion
      required Widget Function(BuildContext, int) itemBuilder,}) {
```